docs/helpers/with.md

```diff
@@ -19,6 +19,22 @@ Renders `BLOCK` with the result of `EXPRESSION` added to the top of the [can-vie
 @param {can-stache.sectionRenderer} BLOCK A template that is rendered
 with the context of the `EXPRESSION`'s value.
 
+@signature `{{#with HASHES}}BLOCK{{/with}}`
+
+Renders `BLOCK` with the key-value pairs from a [can-stache/expressions/hash] added to the top of the [can-view-scope].
+
+```
+{{#with _street=person.address.street _city=person.address.city}}
```

Comment:
I don't think we should show these with `_`s in the examples. It might give people the impression that `_` does something special.